easybuild/main.py

```diff
@@ -474,6 +474,8 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
             ordered_ecs = easyconfigs
     elif pr_options:
         ordered_ecs = None
+    elif options.upload_test_report:
+        raise EasyBuildError("No easyconfigs left to be built (use --rebuild?)")
```

Comment:
Hmm, I wonder if we should somehow allow uploading the test report anyway (but `--force` won't work for this...).

Even if all easyconfigs are already installed, there are exceptional use cases where it may be useful to upload a test report (to communicate system info and EasyBuild configuration, for example)

Reply:
That could work, yes...
That would involve making `--upload-test-report` a `store_or_None` option, so you can allow using it both with and without passing a value; see also `--inject-checksums` or `--force-download` for example.

Are you up for looking into this yourself?

Reply:
when I actually have time again, sure :)